pgjdbc/src/main/java/org/postgresql/Driver.java

```diff
@@ -645,9 +645,10 @@ public boolean jdbcCompliant() {
   private static HostSpec[] hostSpecs(Properties props) {
     String[] hosts = castNonNull(props.getProperty("PGHOST")).split(",");
     String[] ports = castNonNull(props.getProperty("PGPORT")).split(",");
+    String localSocketAddress = props.getProperty("localSocketAddress");
```

Comment:
This should not be a string literal for the property name. It should be a value in the PGProperty enum.

The PGHOST / PGPORT should likely be cleaned up as well but I'd keep that separate for now.

Reply:
hmmm I should have never merged this like this.